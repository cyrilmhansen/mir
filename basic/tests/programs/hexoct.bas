10 PRINT &H1A
20 PRINT &O10
30 PRINT &O377
40 PRINT &Hff
50 END
