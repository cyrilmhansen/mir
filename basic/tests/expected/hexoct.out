26
8
255
255
